src/tests/unit/gameSlice.test.ts

```diff
@@ -0,0 +1,65 @@
+import gameSlice from '../../store/slices/gameSlice'
+import gridsFixtures from '../fixtures/grids.json'
+
+describe('store/slices/gameSlice', () => {
+  it('reduces state to initialized a grid with specific width and height', () => {
```

Comment:
pas hyper clair le nom de ce test... 
`it('initializes a grid based on given height and width')` ou plus simplement `it('reduces the initialize action')`